test/programs/DiagrammaticPrograms.jl

```diff
@@ -0,0 +1,816 @@
+module TestDiagrammaticPrograms
```

Comment:
This whole file has been moved to DataMigrations.jl and seems to have been accidentally added here.